test/WormholeReceiver.t.sol

```diff
@@ -85,3 +87,18 @@ contract IsRegisteredSender is WormholeReceiverTest {
     receiver.isRegisteredSenderModifierFunc(sourceChain, senderBytes);
   }
 }
+
+contract ReplayProtect is WormholeReceiverTest {
+  function testFuzz_SuccessfullyReceiveMessage(bytes32 deliveryHash) public {
+    receiver.replayProtectModifierFunc(deliveryHash);
+  }
```

Comment:
This test feels redundant, if the below test passes this one will always pass

Reply:
Good point

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/39/commits/ca5bd0c521443aff2e0975e30b77fa818b97ac90